src/sentry/api/serializers/models/group.py

```diff
@@ -219,6 +232,7 @@ def get_attrs(self, item_list, user):
                 'resolution': resolution,
                 'resolution_actor': resolution_actor,
                 'share_id': share_ids.get(item.id),
+                'times_seen': times_seen.get(item.id, 0),
```

Comment:
I don't love that this will return a result if there is an issue where a key that should be in the mapping is missing due to a bug

Reply:
Why not just `KeyError` then?